app/src/util/form-schema/index.ts

```diff
@@ -57,29 +57,33 @@ export interface SuggestedActivity {
 }
 
 export interface Methodology {
-  id: string;
-  disabled?: boolean;
-  activities?: Activity[];
-  inputRequired?: string[];
-  formula?: string;
-  fields?: any[];
-  suggestedActivities?: SuggestedActivity[];
-  suggestedActivitiesId?: string;
+    id: string;
+    disabled?: boolean;
+    activities?: Activity[];
+    inputRequired?: string[];
+    formula?: string;
+    fields?: any[];
+    suggestedActivities?: SuggestedActivity[];
+    suggestedActivitiesId?: string;
+    activityTypeField?: string,
+    activityUnitsField?: string,
 }
 
 export interface DirectMeasure {
-  id: string;
-  suggestedActivitiesId?: string;
-  suggestedActivities?: Activity[];
-  inputRequired?: string[];
-  "extra-fields"?: ExtraField[];
+    id: string;
+    suggestedActivitiesId?: string;
+    suggestedActivities?: Activity[];
+    inputRequired?: string[];
+    "extra-fields"?: ExtraField[];
```

Comment:
Can we use a JS-compatible camelCased name here for consistency? Or is there no other way because you're doing a lookup somewhere?

Reply:
this is the interface for what's defined in the json so I can't change it